CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 This is a list of high-level changes for each release of AWX. A full list of commits can be found at `https://github.com/ansible/awx/releases/tag/<version>`.
 
+## 13.0.0 (Jun 19, 2020)
```

Comment:
Let's aim for maybe Monday on this, I know we're likely to merge @jrb's work tomorrow and it include it in this list.

Reply:
I've updated along with mentioning import/export.